packages/cairo.rb

```diff
@@ -7,28 +8,14 @@ class Cairo < Package
   source_url 'https://www.cairographics.org/releases/cairo-1.14.12.tar.xz'
```

Comment:
Please bump the version to detect the changes, otherwise `crew update` will not update previous installs of cairo.

Reply:
How to do that?

Reply:
`version '1.14.12-1'` should do it.